test/reddex/reports/report_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Reddex.ReportTest do
+  use Reddex.DataCase
+
+  alias Reddex.{Report, ReportItem}
```

Comment:
Most of the time you are using the multiple single line alias/require/import/use directives but here you are using the multi-alias/require/import/use syntax